dealroom_firestore_connector/__init__.py

```diff
@@ -70,6 +86,7 @@ def set(doc_ref, *args, **kwargs):
     """
     try:
         doc_ref.set(*args, **kwargs, merge=True)
+        _update_last_edit(doc_ref)
```

Comment:
This might give problems: according to [firestore best practices for Read and Write operations](https://cloud.google.com/firestore/docs/best-practices#read_and_write_operations):

Pay special attention to the **1 write per second limit** for documents and the limit of 1,000,000 concurrent connections per database. These are soft limits that Firestore does not stop you from exceeding. However, going over these limits might affect performance, depending on your total read and write rates.

Reply:
Could be yes, but don't we have this issue anyways with all the ops we do on Firestore? This for sure affects general performance but it's a requirement